crates/subspace-service/src/lib.rs

```diff
@@ -124,6 +127,26 @@ pub type FraudProofVerifier<RuntimeApi, ExecutorDispatch> = subspace_fraud_proof
     Hash,
 >;
 
+/// Subspace networking instantiation variant
+#[derive(Debug, Clone)]
+#[allow(clippy::large_enum_variant)]
+pub enum SubspaceNetworking {
+    /// Use existing networking instance
+    Reuse {
+        /// Node instance
+        node: Node,
+        /// Bootstrap nodes used (that can be also sent to the farmer over RPC)
+        bootstrap_nodes: Vec<Multiaddr>,
```

Comment:
I wonder, why we need bootstrap nodes additionally here. Don't we have them internally within the `Node` instance?

Reply:
Maybe we do, maybe we do not. Either way, it is an implementation detail and these nodes are sent over RPC to the connected farmer so that farmer can bootstrap themselves onto the DSN. Exposing bootstrap nodes on `Node` just for this single use case felt awkward.